manifests/istio-control/istio-discovery/templates/configmap-jwks.yaml

```diff
@@ -1,4 +1,3 @@
-{{ if or (eq .Values.revision "") (not .Values.clusterResources) }}
```

Comment:
I'm not sure how we should handle this resource - seems like a user-configurable setting that is better handled post-install. We need to keep it for the default install, but for other revisions it may be better to not install it, and instead document how the user should add/modify this as needed at runtime ( like any other configuration )

Reply:
Should we just make this always mounted as optional, then user can directly make a config map maybe?

Actually now that i am looking at this, this is 100% broken, we create a volume but its never mounted. @yangminzhu is anyone actually using it?